packages/atomic/src/locales.json

```diff
@@ -1341,5 +1341,11 @@
   "smart-snippet-people-also-ask": {
     "en": "People also ask",
     "fr": "Autres questions posées"
+  },
+  "edit-insight": {
+    "en": "Edit insight"
+  },
+  "insight-history": {
+    "en": "Insight action history"
```

Comment:
was gonna propose something but I'm not sure of that feature name in french 😄

Reply:
insightés